### PR TITLE
atarist.cpp: filled in keyboard input matrix so that symbol keys work

### PR DESCRIPTION
**Pull request by goldnchild**
I wanted to use DR-LOGO on the Atari ST driver, but couldn't type the [ and ] symbols, and in fixing that, noticed that many symbols did not work because the matrix hasn't been filled in.

Also some of the symbols with SHIFT+8,9,0 weren't correct (at least for US keyboard)

Using a graphic from https://forum.atari-home.de/gallery/3282_20_07_12_2_00_46.jpg as a reference.

### CODE REVIEW

**Comment by cuavas** [comment]
@galibert is refactoring this driver at the moment.  It’s going to be changed completely fairly soon.  It’s not worth doing conflicting minor changes like this when the driver is in its current sorry state.

File: src/mame/atari/atarist.cpp [hunk 1]
```diff
@@ -1740,7 +1740,7 @@ static INPUT_PORTS_START( ikbd )
 	PORT_BIT( 0x08, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_NAME("Tab") PORT_CODE(KEYCODE_TAB) PORT_CHAR('\t')
 	PORT_BIT( 0x10, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_CODE(KEYCODE_Q) PORT_CHAR('Q')
 	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_CODE(KEYCODE_A) PORT_CHAR('A')
-	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_KEYBOARD )
+	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_CODE(KEYCODE_PGDN) // ISO KEY
```
Comment (cuavas) [inline src/mame/atari/atarist.cpp:1743]: If by “ISO KEY” you mean the key to the right of left shift, you’re supposed to use `KEYCODE_BACKSLASH2` for that.

Comment (cuavas) [reply src/mame/atari/atarist.cpp:1743]: `IPT_KEYBOARD` inputs need names or characters to represent the keycap markings.

Comment (goldnchild) [reply src/mame/atari/atarist.cpp:1743]: > @galibert is refactoring this driver at the moment. It’s going to be changed completely fairly soon. It’s not worth doing conflicting minor changes like this when the driver is in its current sorry state.

Ok, I understand.  Would you like me to close the PR then?

Comment (angelosa) [reply src/mame/atari/atarist.cpp:1743]: You have to target his branch if really ...

**Pull request closed without merging**